README.md

```diff
@@ -61,6 +61,8 @@ dsc_windowsfeature {'IIS':
 
 All DSC Resource names and parameters have to be in lowercase, e.g: `dsc_windowsfeature` or `dsc_name`.
 
+When it comes to `ensure =>` (Puppet's `ensure`) versus `dsc_ensure =>` (DSC's `Ensure`), you can use one or the other in your manifests for Puppet DSC resource types. If you use both in a Puppet DSC resource, `dsc_ensure` will override the value in `ensure`, so the value for `ensure` will essentially be ignored. We recommend you pick whichever one is comfortable for you. If you have no preference, use `dsc_ensure` as it is a closer match for converting from the DSC properties to Puppet DSC resources and is the overriding type so there is less confusion if both are accidentally included.
```

Comment:
@cowofevil discovered this isn't quite true - see https://tickets.puppetlabs.com/browse/MODULES-2267